kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Scheduler.kt

```diff
@@ -1,17 +1,22 @@
+@file:JvmMultifileClass
```

Comment:
Most of your changes in this file involve these annotations, making the file harder to read, or replacing logger creations. I don't see the need or point of those changes, and they're out of scope of the PR.

I support the change to `forEach` for the jobs, though.